App.js

```diff
@@ -29,12 +31,19 @@ export default class App extends Component {
     })
   }
 
+  _getUser (user) {
+    this.setState({
+      username: user.username
+    })
+  }
+
   render () {
     return (
       <View style={styles.app}>
         <HeaderApp userLogged={this.state.userLogged} />
+        <Text>{this.state.username}</Text>
         <View>
-          <LoginForm />
+          <LoginForm _getUser={this._getUser} />
```

Comment:
Can't you just pass `user={ this.user }`?
The component logic seems broken to me. I'm no expert to React, but it reads weird that the get sets a state that actually is a json converter within the component